specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/examples/ValidateTransfer.json

```diff
@@ -0,0 +1,41 @@
+{
+  "parameters": {
+    "api-version": "2019-10-01-preview",
+    "transferName": "transferName",
+    "parameters": {
+      "properties": {
+        "productDetails": [
+          {
+            "productType": "AzureSubscription",
+            "productId": "subscriptionId"
+          },
+          {
+            "productType": "AzureReservation",
+            "productId": "reservedInstanceId"
+          }
+        ]
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "properties": {
+              "productId": "subscriptionId",
+              "status": "Succeeded",
```

Comment:
Is this a valid example? Success status with error result. Ideally examples represent real world interaction, not just an example of the schema.

Reply:
Thanks. 
Fixed the example.